deploy/template.yaml

```diff
@@ -1036,6 +1037,7 @@ Resources:
         RequestParameters:
           integration.request.path.proxy: "method.request.path.proxy"
           integration.request.header.x-forwarded-for: "method.request.header.x-forwarded-for"
+          integration.request.header.x-forwarded-host: "method.request.header.host"
```

Comment:
This adds an `x-forwarded-host` header on the response, with the value of the `host` header